README.md

```diff
@@ -32,6 +32,57 @@ To uninstall TinyTeX, use the command:
 choco uninstall tinytex
 ```
 
-## Scoop package
+## Scoop packages
 
-Coming...
+Scoop is another package manager for Windows. You need to install [_scoop_](https://scoop-docs.now.sh/docs/getting-started/Quick-Start.html) first to use it from powershell.
+
+Apps for _scoop_ are available through "buckets". For now, TinyTex binary packages are not available in the default **main** or **extras** buckets that comes with a new installation of _scoop_, but through the scoop bucket **r-bucket** at https://github.com/cderv/r-bucket/ with other R-related tools.
```

Comment:
`TinyTex` -> `TinyTeX` ([I have to say this every time](https://yihui.org/en/2017/12/typing-names/) 😂 )

Reply:
oups 

I made the change!